File: test/fixtures/ruby/person.rb
def add(x, y)
  x + y
end

class Person
  attr_reader :name, :age

  def initialize(name, age)
    @name = name
    @age = age
  end

  def greet
    "Hello, #{@name}"
  end
end

if __FILE__ == $0
  p = Person.new("Alice", 30)
  puts p.greet
  result = add(1, 2)
  puts result
end

File: test/fixtures/c/person.c
#include "person.h"

Person create_person(const char *name, int age) {
  Person p;
  p.name = (char *)name;
  p.age = age;
  return p;
}

int add(int x, int y) { return x + y; }

int main(void) {
  Person p = create_person("Alice", 30);
  return add(1, 2);
}
